### PR TITLE
Packages required by XRays module

### PR DESCRIPTION
**Pull request by sanguinariojoe**
Thank you for creating a pull request to contribute to FreeCAD! To ease integration, we ask you to conform to the following items. Pull requests which don't satisfy all the items below might be rejected. If you are in doubt with any of the items below, don't hesitate to ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10)!

- [X]  Your pull request is confined strictly to a single module. That is, all the files changed by your pull request are either in `App`, `Base`, `Gui` or one of the `Mod` subfolders. If you need to make changes in several locations, make several pull requests and wait for the first one to be merged before submitting the next ones
- [X]  In case your pull request does more than just fixing small bugs, make sure you discussed your ideas with other developers on the FreeCAD forum
- [X]  Your branch is [rebased](https://git-scm.com/docs/git-rebase) on latest master `git pull --rebase upstream master`
- [X]  All FreeCAD unit tests are confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [X]  All commit messages are [well-written](https://chris.beams.io/posts/git-commit/) ex: `Fixes typo in Draft Move command text`
- [X]  Your pull request is well written and has a good description, and its title starts with the module name, ex: `Draft: Fixed typos`
- [X]  Commit messages include `issue #<id>` or `fixes #<id>` where `<id>` is the issue ID number from our [Issues database](https://github.com/FreeCAD/FreeCAD/issues) in case a particular commit solves or is related to an existing issue. Ex: `Draft: fix typos - fixes #4805`

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.20 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=56135).

---

I just need a bunch of Python packages in my X-Rays simulator:

 - beautifulsoup4 (It is an ancient and popular package, zero risk associated)
 - openexr (It is now under my control, so no risk again)
 - qtrangeslider (The developer seems pretty active in the community. However, this is a small project, so it would imply some risks)
 - scikit-image (Another super-popular package)
 - trimesh (This tool has become a really handy and popular one in Python. I have been using this for years without much complains)

### CODE REVIEW

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6763">pipeline status</a> for feature branch [PR_6763](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6763). Pipeline [518675630](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/518675630) was triggered at [fdcd2b2](https://github.com/FreeCAD/FreeCAD/pull/6763/commits/fdcd2b2b3881309f7b3958768c2bf04013bbac40). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/AddonManager/ALLOWED_PYTHON_PACKAGES.txt [hunk 1]
```diff
@@ -18,6 +19,7 @@ msgpack
 numpy
 ocp
 olefile
+openexr
```
Comment (chennes) [inline src/Mod/AddonManager/ALLOWED_PYTHON_PACKAGES.txt:22]: Is this the right case? Pypl lists it as "OpenEXR"

**Comment by chennes** [comment]
Obviously the actual OpenEXR library will still have to be installed manually by the user, but otherwise I think these are all fine. @donovaly did you see the discussion in the forums about moving this requirements file out of FreeCAD proper and into the FreeCAD-Addons repo, so we can update it independent of FreeCAD releases?

**Comment by donovaly** [comment]
> ETA: Here it is https://forum.freecadweb.org/viewtopic.php?f=10&t=68089

Fine with me.

**Comment by chennes** [comment]
This is now obsolete, I have migrated this file to the FreeCAD-addons GitHub repo (and included these packages in it).

**Pull request closed without merging**